webapp-django/crashstats/crashstats/management/commands/archivescraper.py

```diff
@@ -254,7 +258,11 @@ def scrape_candidate_version(self, link, product_name, final_releases):
             # directories
             for json_link in json_links:
                 json_file = self.download(json_link)
-                data = json.loads(json_file)
+                try:
+                    data = json.loads(json_file)
+                except json.decoder.JSONDecodeError:
+                    self.worker_write(f"not valid json: {json_link}")
+                    continue
```

Comment:
I occasionally get errors where the returned content isn't JSON. We should probably fix something to retry this situation a few times in case it's an ephemeral server error. I'm going to leave it like this for now.